modules/dataframe/include/inviwo/dataframe/datastructures/dataframe.h

```diff
@@ -288,64 +300,22 @@ struct DataTraits<DataFrame> {
-                                using PrecisionType = typename util::value_type<ValueType>::type;
-
-                                const auto& vec = brprecision->getDataContainer();
-
-                                if (vec.empty()) {
-                                    return {"-", "-"};
-                                }
-
-                                auto createSS = [](const PrecisionType& min,
-                                                   const PrecisionType& max)
-                                    -> std::pair<std::string, std::string> {
-                                    if constexpr (std::is_floating_point_v<ValueType>) {
-                                        std::stringstream minSS;
-                                        std::stringstream maxSS;
-
-                                        minSS << std::defaultfloat << min;
-                                        maxSS << std::defaultfloat << max;
-
-                                        return {minSS.str(), maxSS.str()};
-                                    } else {
-                                        return {std::to_string(min), std::to_string(max)};
-                                    }
-                                };
-
-                                if constexpr (util::extent<ValueType>::value == 1) {
-                                    auto [min, max] =
-                                        std::minmax_element(std::begin(vec), std::end(vec));
-
-                                    // call lambda
-                                    return createSS(*min, *max);
-                                } else {
-                                    auto min{std::numeric_limits<PrecisionType>::max()};
-                                    auto max{std::numeric_limits<PrecisionType>::min()};
-
-                                    std::for_each(std::begin(vec), std::end(vec),
-                                                  [&min, &max](const ValueType& v) {
-                                                      min = std::min(min, glm::compMin(v));
-                                                      max = std::max(max, glm::compMax(v));
-                                                  });
-
-                                    return createSS(min, max);
-                                }
-                            });
-
-                tb2(std::to_string(i + 1),
-                    data.getColumn(i)->getBuffer()->getDataFormat()->getString(),
-                    data.getColumn(i)->getBuffer()->getSize(), data.getHeader(i),
-                    range(col_q.get()), minString, maxString);
+                auto [minString, maxString] = [&]() -> std::pair<std::string, std::string> {
+                    if (col_q->getSize() == 0) return {"-", "-"};
+
+                    const auto minmax = col_q->getDataRange();
+                    return {fmt::format("{:8.4g}", minmax.x), fmt::format("{:8.4g}", minmax.y)};
```

Comment:
is this equivalent to std::defaultfloat?

Reply:
what does std::defaultfloat? even do? seems like some reset?

Reply:
Sets the floatfield of the stream str to zero, as if by calling str.unsetf(std::ios_base::floatfield). This enables the default floating-point formatting, which is different from fixed and scientific.

See https://en.cppreference.com/w/cpp/io/manip/fixed

```
┌──────────┬────────────┬──────────────────────────┐
│  number  │   iomanip  │      representation      │
├──────────┼────────────┼──────────────────────────┤
│ 0.0      │ fixed      │ 0.000000                 │
│ 0.0      │ scientific │ 0.000000e+00             │
│ 0.0      │ hexfloat   │ 0x0p+0                   │
│ 0.0      │ default    │ 0                        │
├──────────┼────────────┼──────────────────────────┤
│ 0.01     │ fixed      │ 0.010000                 │
│ 0.01     │ scientific │ 1.000000e-02             │
│ 0.01     │ hexfloat   │ 0x1.47ae147ae147bp-7     │
│ 0.01     │ default    │ 0.01                     │
├──────────┼────────────┼──────────────────────────┤
│ 0.00001  │ fixed      │ 0.000010                 │
│ 0.00001  │ scientific │ 1.000000e-05             │
│ 0.00001  │ hexfloat   │ 0x1.4f8b588e368f1p-17    │
│ 0.00001  │ default    │ 1e-05                    │
└──────────┴────────────┴──────────────────────────┘
```

Reply:
Same as g I think